### PR TITLE
Bump jellyfin-web to 10.6.0

### PR DESCRIPTION
**Pull request by Maxr1998**
Compiled and tested this locally, everything seems to work just fine (~~nvm, notification is obviously broken. I'm gonna look into it I guess~~ - fixed).

### CODE REVIEW

File: src/NativeShell/src/RemotePlayerService.java [hunk 1]
```diff
@@ -80,12 +80,12 @@ public void onReceive(Context context, Intent intent) {
     };
 
     private static void sendCommand(String action) {
-        String url = "javascript:require(['inputManager'], function(inputManager){inputManager.trigger('" + action + "'" + ");});";
+        String url = "javascript:require(['inputManager'], function(inputManager){inputManager.handleCommand('" + action + "'" + ");});";
```
Comment (thornbill) [inline src/NativeShell/src/RemotePlayerService.java:83]: Was this change included in 10.6? It seems to have just been merged earlier today. https://github.com/jellyfin/jellyfin-web/pull/1593

Comment (dmitrylyzo) [reply src/NativeShell/src/RemotePlayerService.java:83]: This PR uses jellyfin/jellyfin-web@b3fb64f114b7ff4c3a4cd068460c7d7b4df745a3 from `master` branch.

Comment (dkanada) [reply src/NativeShell/src/RemotePlayerService.java:83]: Ah, we agreed to use the stable versions only for the time being.

Comment (Maxr1998) [reply src/NativeShell/src/RemotePlayerService.java:83]: Ah, my mistake, I was also testing another PR to the web repo while noticing the issue. Should've checked when the change was committed before. Gonna revert that fix then, gimme a second.

Comment (dmitrylyzo) [reply src/NativeShell/src/RemotePlayerService.java]: > This PR uses [jellyfin/jellyfin-web@b3fb64f](https://github.com/jellyfin/jellyfin-web/commit/b3fb64f114b7ff4c3a4cd068460c7d7b4df745a3) from `master` branch.

Forget what I said - commit is OK. _It would be better if I looked on normal git tree, not GH._

**Pull request merged**